providers/shared/components/cache/id.ftl

```diff
@@ -49,6 +49,32 @@
                     }
                 ]
             },
+            {
```

Comment:
Feels like this should be an attribute set given it is turning up in multiple places.

Reply:
Done but left the queuehost which has a default as an explicit definition

Reply:
You can override the default attributeset with a default value in the queuehost.